addons/account/views/account_onboarding_templates.xml

```diff
@@ -166,14 +166,26 @@
             <form>
                 <div class="mb16">Choose a default sales tax for your products.</div>
                 <label for="account_sale_tax_id" string="Sales Tax"/>
-                <field name="account_sale_tax_id" />
+                <field name="account_sale_tax_id" domain="[('type_tax_use','=','sale')]" options="{'no_open': 1}"
+                       context="{'default_type_tax_use': 'sale', 'form_view_ref': 'account.view_tax_form_sale_inherited'}"/>
                 <footer>
-                    <button string="Apply" class="btn btn-primary" type="object" name="action_save_onboarding_sale_tax" />
+                    <button string="Apply" class="btn btn-primary" type="object" name="action_save_onboarding_sale_tax"/>
                     <button string="Cancel" class="btn-secondary" special="cancel" />
                 </footer>
             </form>
         </field>
     </record>
+    <record id="view_tax_form_sale_inherited" model="ir.ui.view">
```

Comment:
why not do it into the view directly... you are in the same module...

Reply:
The parent view is used elsewhere, and we don't want to hide the field everywhere? Maybe there is another way that I am not aware of.